### PR TITLE
Add XGeo-1.0 package in conda-forge

### PR DESCRIPTION
**Pull request by Geosynopsis**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vendor other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

File: recipes/xgeo/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  script:  "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [(py<36)]
+
+requirements:
+  host:
+    - python
+    - pip
+  run:
+    - python
+    - pandas>=0.24.1
```
Comment (chrisburr) [inline recipes/xgeo/meta.yaml]: A space is needed here before the `>=` (but not after)
```suggestion
    - pandas >=0.24.1
```
The same applies to the other dependencies.

File: recipes/xgeo/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  version: {{ version }}
+
+source:
+  url: https://github.com/Geosynopsis/xgeo/archive/xgeo-v{{ version }}.tar.gz
+  sha256: 2b494a56c8f91967828c49806a0e8cdde2f4af8c24bc2b2ede509ce6ff96a7dc
+
+build:
+  number: 0
+  script:  "{{ PYTHON }} -m pip install . --no-deps -vv"
+  skip: True  # [(py<36)]
```
Comment (chrisburr) [inline recipes/xgeo/meta.yaml]: This looks like it could be `noarch`?
```suggestion
  noarch: python
```

File: recipes/xgeo/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+
+requirements:
+  host:
+    - python
```
Comment (chrisburr) [inline recipes/xgeo/meta.yaml]: If it becomes noarch you can keep the Python version constraint with:
```suggestion
    - python >=3.6
```

File: recipes/xgeo/meta.yaml [hunk 4]
```diff
@@ -0,0 +1,43 @@
...
+    - pip
+  run:
+    - python
```
Comment (chrisburr) [inline recipes/xgeo/meta.yaml]: If it becomes noarch you can keep the Python version constraint with:
```suggestion
    - python >=3.6
```

File: recipes/xgeo/meta.yaml [hunk 5]
```diff
@@ -0,0 +1,43 @@
...
+    - shapely>=1.6.4
+
+test:
+  imports:
+    - xarray
+    - xgeo
+
+about:
+  home: https://geosynopsis.github.io/xgeo
+  license: Apache 2.0
```
Comment (chrisburr) [inline recipes/xgeo/meta.yaml]: From https://opensource.org/licenses/alphabetical:
```suggestion
  license: Apache-2.0
```

File: recipes/xgeo/LICENSE.txt [hunk 6]
```diff
@@ -0,0 +1,201 @@
+                                 Apache License
```
Comment (chrisburr) [inline recipes/xgeo/LICENSE.txt]: This file isn't needed as it's included in the source tarball.

**Comment by Geosynopsis** [comment]
@chrisburr Thanks for the suggestions.

**Pull request merged**